impl/src/main/java/org/jboss/cdi/tck/tests/alternative/selection/priority/AltBeanProducingAlternative.java

```diff
@@ -0,0 +1,24 @@
+package org.jboss.cdi.tck.tests.alternative.selection.priority;
+
+import jakarta.annotation.Priority;
+import jakarta.enterprise.inject.Alternative;
+import jakarta.enterprise.inject.Produces;
+
+@Alternative
+@Priority(1)
+public class AltBeanProducingAlternative {
```

Comment:
This class is actually incorrect - it misses bean defining annotation.
I'll force push a fix in a bit.